apps/sms/views/conversation/js/compose.js

```diff
@@ -406,6 +406,14 @@ var Compose = (function() {
         subject.show();
       }
 
+      if (!draft.content) {
```

Comment:
yeah because initially the function was thought as "initialize the composer, and if there is a draft load it".